docs/ja/user-guide/metadata.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: main
+title: Metadata
+category: User Guide
+menu: menu
+toc:
+- title: Metadata
+  url: "#metadata"
+- title: What is Metadata?
```

Comment:
We should also translate these titles and urls.

Reply:
I see. I also translate this to Japanese.